server/routes/api/auth.js

```diff
@@ -0,0 +1,68 @@
+const express = require("express");
+const router = express.Router();
+const isEmail = require('validator/lib/isEmail');
+const normalizeEmail = require('validator/lib/normalizeEmail');
+const bcrypt = require("bcryptjs");
+const config = require("config");
+const jwt = require("jsonwebtoken");
+const auth = require('../../middleware/auth');
+
+const User = require("../../models/User");
+
+// @route   POST api/auth
+// @desc    Auth user
+// @access  Public
+router.post("/", function(req, res, next) {
+  const { email, password } = req.body;
+  try {
+    const normalizedEmail = normalizeEmail(email);
+    if (!isEmail(normalizedEmail)) throw new Error('The email you entered is invalid');
+    if (!password || !email) throw new Error('Missing field(s)');
+    const date = new Date();
```

Comment:
is this used?

Reply:
Yeah, this is for input validation for a login page

Reply:
Oh but I'll remove the `date`, which is not used